src/compiler/transformers/stencil-import-path.ts

```diff
@@ -28,7 +29,7 @@ export const serializeImportPath = (data: SerializeImportData, styleImportData:
       p = './' + p;
     }
 
-    if (styleImportData === 'queryparams' || styleImportData === undefined) {
+    if (moduleSystem !== 'cjs' && (styleImportData === 'queryparams' || styleImportData === undefined)) {
```

Comment:
It depends I guess. Using query params in require calls is actually not supported and would fail, e.g.

CJS:
```ts
const bar = require('./b?foo=bar')
// throws `Error: Cannot find module './b?foo=bar'`
```

ESM:
```ts
import b from './b.js?foo=bar'
// works 👍
```

Now, I can imagine that there are use cases where folks compile it into CJS which then gets picked up by another bundler that compiles it back in ESM. If we assume that the compiler respects the require params and ports them along than it could introduce issues.

Reply:
ok cool, I'm not worried then!